library/core/class.controller.php

```diff
@@ -2142,4 +2164,23 @@ public function title($Title = null, $Subtitle = null) {
 
         return $this->data('Title');
     }
+
+    /**
+     * Set the destination URL where the page will be redirected after an ajax request.
+     *
+     * @param string|null $destination Destination URL or path.
+     *      Redirect to current URL if nothing or null is supplied.
+     * @param bool $trustedOnly Non trusted destinations will be redirected to /home/leaving?Target=$destination
+     */
+    public function redirectTo($destination = null, $trustedOnly = true) {
```

Comment:
I thought about it but the property if actually protected so it can only be confusing internally.
I also wanted the function to have the same name than it's non AJAX counterpart.